input/intro-notes/StructureDefinition-ehealth-servicerequest-intro.md

```diff
@@ -12,3 +12,6 @@ The eHealth profile of ServiceRequest has the following extensions:
 * `ehealth-sharingApprovalPolicy` Selects whether the approval of publication to national document sharing of measurements (resource(s) resulting from conducting the activity) must be done manually or if it is done automatically. The initial value will be inherited from the ActivityDefinition referenced in `definition`, but can subsequently be adapted for the particular Patient by adjusting the ServiceRequest.
 * `ehealth-servicerequest-statusHistory` contains history of changes in `status`, and is automatically maintained by the infrastructure.
 * `ehealth-servicerequest-statusSchedule` contains a list of scheduled changes in `status`. The infrastructure applies these changes automatically, and will remove the entry for the change once it has been applied.
+
+### Occurrence validity
+The `ehealth-servicerequest-occurrence[x]` field becomes both mandatory and must be valid when a ServiceRequest is marked as `active`, `on-hold`, or `completed`, as specified by `ehealth-servicerequest-status` or `ehealth-servicerequest-statusSchedule`. This validation applies when updating either the CarePlan (if `ehealth-careplan-status` or `ehealth-careplan-statusSchedule` is `active`) or the ServiceRequest.
```

Comment:
The validity depends on the variant of occurrence[x] as follows:

- `occurrenceDateTime`: Shall contain a date-time with at least hours and minutes (and hence timezone)
- `occurrencePeriod`: Shall contain at least a `Period.start` containing a date-time with at least hours and minutes (and hence timezone).
- `occurrenceTiming`: Shall either be ad-hoc or recurring as follows:
-- `occurrenceTiming` (ad-hoc): **<Oana, what are the rules here?>**
-- `occurrenceTiming` (recurring): the elements of `Timing.repeat` shall adhere to:
--- `boundsPeriod` shall be specified and shall contain at least `start` containing a date-time with at least hours and minutes (and hence timezone). If present, `end` shall contain a date-time with at least hours and minutes (and hence timezone) which shall be equal to or after `start`
--- `dayOfWeek`: optional. When one or more values specified, `period` and `periodUnit` shall specify either no value (being empty), 1 d (one day), pos-integer wk (week)
--- `timeOfDay`: optional. When one or more values specified, `period` and `periodUnit` shall specify either no value (being empty), pos-integer d (day), pos-integer wk (week), pos-integer mo (month), or pos-integer a (year)
--- `durationUnit`: required when `duration` is specified. Shall be one of s, min, h, d, wk, a
--- `period`: optional. When empty, it corresponds to occurrence every one day (every day)
--- `periodUnit`: required when `period` is not empty. Shall be one of min, h, d, wk, mo, a

See the wiki for examples.